ReSharper.FSharp/src/FSharp.Psi.Features/src/QuickFixes/ReplaceWithAssignmentExpressionFix.fs

```diff
@@ -0,0 +1,27 @@
+﻿namespace JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.QuickFixes
+
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.Highlightings.CommonErrors
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Parsing
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Tree
+open JetBrains.ReSharper.Resources.Shell
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Util
+open JetBrains.ReSharper.Psi.Tree
+
+type ReplaceWithAssignmentExpressionFix(warning: UnitTypeExpectedWarning) =
+    inherit FSharpQuickFixBase()
+   
+    let expr = warning.Expr.As<IBinaryAppExpr>()
+    override x.IsAvailable _ =        
+        if not (isValid expr) || expr = null then false else
```

Comment:
What does `expr = null` check here?

Reply:
> What does `expr = null` check here?

I forgot that is valid already checks this